contributing_to_docs/doc_guidelines.adoc

```diff
@@ -863,21 +863,71 @@ Text for admonition
 [id="api-object-formatting"]
 == API object formatting
 
-Use initial capitalization and upper camel case
-(link:https://en.wikipedia.org/wiki/Camel_case#Variations_and_synonyms[PascalCase])
-for Kubernetes/OpenShift API objects and do not mark them up unless referring to
-a specific field or variable name from a spec or manifest.
+For terms that are API objects, the way they are written depends on whether the term is a general reference or an actual reference to the object.
 
-This matches general Kubernetes usage and makes it obvious that a specific
-concept is being referred to. For example:
+[id="api-object-general-references"]
+=== General references
 
-- Pod
-- Deployment
-- Operator
-- DaemonSet (not "daemon set", "daemonset", "daemonSet", or "Daemonset")
+When referring to API objects in general usage, use lowercase and separate multi-word API objects. By general reference, this means any time you are speaking conceptually, or generally, about these components in a cluster. *Default to following this guidance unless you are specifically interacting with/referring to the API object.*
+
+For example:
+
+* pod
+* node
+* daemon set
+* config map
+* deployment
+* deployment config
+* image stream
+* persistent volume claim
+
+.Examples of general references
+....
+Kubernetes runs your workload by placing containers into pods to run on nodes.
+
+You must have at least one secret, config map, or service account.
+
+The total number of persistent volume claims in a project.
+....
+
+Note that if an object uses an acronym or other special capitalization, then its general reference should honor that. For example, general references to `APIService` should be written as "API service", not "api service". Any other exceptions or special guidance are noted in the xref:../../contributing_to_docs/term_glossary.adoc[glossary]. 
```

Comment:
Is `APIService` supposed to be `API Service` here?

Reply:
@lbarbeevargas No, `APIService` is the correct API object kind. If something is monospace and PascalCase, then there should never be a space in there.

You can see the of this particular example in the spreadsheet, line 22: https://docs.google.com/spreadsheets/d/1VWJwd7Yk91CY_BnyCKSpHcPwuRNEYASmxkXRIRp4WEY/edit#gid=0